### PR TITLE
Changed readme to address that source and destnation tables should be…

### PR DESCRIPTION
**Pull request by himanshukohli09**
… in same location while reading from query and views.

Also, clarified the usage of project and dataset options.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -198,10 +198,38 @@ In order to use this feature the following configurations MUST be set:
 * `materializationDataset` must be set to a dataset where the GCP user has table
   creation permission. `materializationProject` is optional.
 
+**Note:** As mentioned [here](https://cloud.google.com/bigquery/docs/writing-results#temporary_and_permanent_tables),
```
Comment (davidrabinowitz) [inline README.md:201]: "here" -> "in the BigQuery documentation"

File: README.md [hunk 2]
```diff
@@ -198,10 +198,38 @@ In order to use this feature the following configurations MUST be set:
...
+the queried tables must be in the same location as the `materializationDataset`.
+Also, if the tables in the `SQL statement` are from projects other than the
+`parentProject` then use the fully qualified table name i.e.
+`[project]:[dataset].[table]`.
```
Comment (davidrabinowitz) [inline README.md:205]: `[project]:[dataset].[table]` -> `[project].[dataset].[table]`

File: README.md [hunk 3]
```diff
@@ -263,15 +291,17 @@ The API Supports a number of options to configure the read
   <tr valign="top">
    <td><code>dataset</code>
    </td>
-   <td>The dataset containing the table.
+   <td>The dataset containing the table. This option is intended to be used only
```
Comment (davidrabinowitz) [inline README.md:294]: This option should be used with standard table and views, but not when loading query results

File: README.md [hunk 4]
```diff
@@ -263,15 +291,17 @@ The API Supports a number of options to configure the read
...
+   with the <code>load()</code>/<code>save()</code> methods.
        <br/>(Optional unless omitted in <code>table</code>)
    </td>
    <td>Read/Write</td>
   </tr>
   <tr valign="top">
    <td><code>project</code>
    </td>
-   <td>The Google Cloud Project ID of the table.
+   <td>The Google Cloud Project ID of the table. This option is intended to be
```
Comment (davidrabinowitz) [inline README.md:303]: This option should be used with standard table and views, but not when loading query results

**Review by davidrabinowitz — CHANGES_REQUESTED** [review]

**Review by himanshukohli09 — COMMENTED** [review]
resolved comments

**Review by davidrabinowitz — APPROVED** [review]

**Pull request merged**